modules/ROOT/pages/installation-prerequisites.adoc

```diff
@@ -43,7 +47,9 @@ If you already have Anypoint Identity Provider, get identity provider data from
 
 === If You Do Not Already Have Anypoint Identity Provider
 
-If you do not already have Anypoint Identity Provider (IdP), configure Salesforce as a SAML IdP:
+If you do not already have Anypoint Identity Provider (IdP), configure Salesforce as a SAML IdP.
```

Comment:
No such product as Anypoint Identity Provider. Perhaps you mean...:
```suggestion
If you do not already have an Anypoint Platform identity provider (IdP), configure Salesforce as a SAML IdP.
```

Reply:
Fixed.